docs/developer-docs/build/using-an-agent.mdx

```diff
@@ -0,0 +1,56 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+# Using an Agent
+
+If you have a canister running, either on your local machine or live on the Internet Computer, you will have two main ways to interact with your canister smart contract. You can talk to the canister using the v2 API using an `agent` that follows the interface specification, or you can use the canister's http interface.
```

Comment:
> or you can use the canister's http interface.

Does this refer to the common practice that implement a `http_request` method in backend canister and use it from frontend?

Reply:
Yes, although it's an official interface via the boundary nodes, so I'm choosing to refer to it as a way that canisters can be interacted with